### ISSUE TITLE
Exception on calling isUpdateAvailable

### ISSUE THREAD

**Issue by rlbjorn** [GigaDroid/velopack_flutter#3]
isUpdateAvailable(url: 'https://test.com');   is throwing the following exception:
```
Exception has occurred.
AnyhowException (AnyhowException(This application is not properly installed: Could not auto-locate app manifest

Stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: <unknown>
  24: BaseThreadInitThunk
  25: RtlUserThreadStart))
```

Context:
On windows, calling isUpdateAvailable after calling Rustlib.init() in debug version. 

Do you have any idea what could cause this?

**Comment by GigaDroid** [comment]
Did you pack it using `vpk pack` and install it using the created installer? If you just run the debug/release build without packaging first, there will be missing files. 

Also, trying to fetch updates from `test.com` will probably throw an exception anyway even if all necessary files are there, as there are no update files located on that domain

**Comment by rlbjorn** [comment]
Oh. I did not pack it. I will try that. Thanks for the information.

**Comment by nrbnlulu** [comment]
Hi, why it is required to "install" the application? (apart from the fact it won't work without it)
is it some constrain on Windows?

**Comment by GigaDroid** [comment]
The installer will place an Update.exe in the directory of the application, which handles fetching and replacing the binaries in case of an update

**Comment by kakaiba-talaga** [comment]
@GigaDroid, please correct me if I am wrong, the packed application has to be installed first before it can be ran in debug mode of your preferred IDE?

Is copying the `Update.exe` in the `~\App\build\windows\x64\runner\Debug` directory not enough?

**Comment by GigaDroid** [comment]
Yes, you should either install the packed application or copy the `Update.exe` into `~\App\build\windows\x64\runner\Debug`. Both ways should work fine.

**Comment by kakaiba-talaga** [comment]
@GigaDroid, copying the `Update.exe` into the `~\App\build\windows\x64\runner\Debug` worked. 🎉

The original error:

> This application is not properly installed: Could not auto-locate app manifest

is now gone. ✅ 

Unfortunately, it introduced another error❗:

> This application is missing a package manifest (.nuspec) or it could not be parsed.

**Comment by GigaDroid** [comment]
Please share the full logs

**Comment by kakaiba-talaga** [comment]
After calling `await isUpdateAvailable(url: updateUrl)`, this is the only error log that I got from it:

```
AnyhowException (AnyhowException(This application is missing a package manifest (.nuspec) or it could not be parsed.

Stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: <unknown>
  24: BaseThreadInitThunk
  25: RtlUserThreadStart))
```

```
#0      SimpleDecoder.decode (package:flutter_rust_bridge/src/codec/base.dart:32:9)
#1      SseCodec._decode (package:flutter_rust_bridge/src/codec/sse.dart:45:55)
#2      SseCodec.decodeObject (package:flutter_rust_bridge/src/codec/sse.dart:35:12)
<asynchronous suspension>
#3      UpdateManagerVelopack._checkForUpdatesMain (package:newhaven/src/utils/update_manager_velopack.dart:112:27)
<asynchronous suspension>
#4      UpdateManagerVelopack.checkForUpdatesOnAppStart.tryCheckForUpdates (package:newhaven/src/utils/update_manager_velopack.dart:188:7)
<asynchronous suspension>
#5      UpdateManagerVelopack.checkForUpdatesOnAppStart (package:newhaven/src/utils/update_manager_velopack.dart:193:5)
<asynchronous suspension>
```

The `update_manager_velopack.dart:112:27` is:

```
thereIsAnUpdate = await isUpdateAvailable(url: updateUrl);
```

**Comment by GigaDroid** [comment]
You also need to move the `sq.version` file into the directory. You can find it in the packaged installation.

**Comment by kakaiba-talaga** [comment]
I copied `sq.version` into the same directory as `Update.exe`, but the same error persists.

```
This application is missing a package manifest (.nuspec) or it could not be parsed.
```

**Comment by GigaDroid** [comment]
I think the `Update.exe` should be in the `/runner`directory and `sq.version` in `/runner/Debug` if I recall correctly. If that doesn't work please consider just using `vpk pack`

**Comment by kakaiba-talaga** [comment]
That didn't work. Moving `Update.exe` to `~\App\build\windows\x64\runner` returned the original error: 

```
This application is not properly installed: Could not auto-locate app manifest
```

I then tried packaging it using `vpk pack ...`. Once installed, it still returned the same error.

```
This application is missing a package manifest (.nuspec) or it could not be parsed.
```

---

What worked is when I copied both `Update.exe` and `sq.version` into `~\App\build\windows\x64\runner` before running `vpk pack ...`. I then ran the installer. The installed app was able to check for an update. 🎉 

Unfortunately, that didn't work for the IDE. It is still returning the error:

```
This application is missing a package manifest (.nuspec) or it could not be parsed.
```
